backend/common/src/main/java/ai/verta/modeldb/common/configuration/EnabledMigration.java

```diff
@@ -19,7 +19,7 @@ private static boolean isMigration() {
             Optional.ofNullable(System.getenv(CommonConstants.ENABLE_LIQUIBASE_MIGRATION_ENV_VAR))
                 .orElse("false"));
     LOGGER.info("ENABLE_LIQUIBASE_MIGRATION_ENV_VAR: {}", enableLiquibaseMigrationEnvVar);
-    return enableLiquibaseMigrationEnvVar;
+    return true;
```

Comment:
Does this need any changes due to the recent migration changes?

Reply:
Reverted